docs/tutorials/add-contracts-pallet.md

```diff
@@ -100,7 +100,7 @@ feature will determine the features that should be used on downstream dependenci
 snippet above should be read as:
 
 > The default feature for this Substrate runtime is `std`. When `std` feature is enabled for the
-> runtime, `parity-scale-codec`, `primitives`, `client`, and all the other listed dependencies
+> runtime, `codec`, `frame-executive`, `frame-support`, and all the other listed dependencies
```

Comment:
We do have [`parity-scale-codec`](https://crates.io/crates/parity-scale-codec), [`primitives`](https://github.com/paritytech/substrate/tree/master/primitives) (when we say primitives, we refer to a category of crates under `primitives` dir, `sp-*` crate packages), and [`clients`](https://github.com/paritytech/substrate/tree/master/client) (`sc-*` crate packages). It will be great if you could rework to link back to the source code directory to help readers understand better. Thanks

Reply:
Indeed, this tutorial needs a full upgrade to 3.0. These changes should not be addressed individually.

Reply:
this was missed in https://github.com/substrate-developer-hub/substrate-developer-hub.github.io/pull/899